iocore/cache/CacheVC.cc

```diff
@@ -21,12 +21,17 @@
   limitations under the License.
  */
 
+#include "I_Cache.h"
 #include "I_CacheDefs.h"
 #include "P_CacheDisk.h"
 #include "P_CacheHttp.h"
+// #include "P_CacheHosting.h"
 #include "P_CacheInternal.h"
 #include "P_CacheVol.h"
 
+// must be included after the others
```

Comment:
Should CacheVC.h include the required files itself?

Reply:
I couldn't get that to work. I spent significantly more time trying to get that to work than I did making the change.